packages/react-components/src/components/CameraButton.tsx

```diff
@@ -149,34 +161,49 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
     strings.tooltipDisabledContent = strings.tooltipVideoLoadingContent;
   }
 
+  const toggleAnnouncerString = useCallback(() => {
+    setAnnouncerString(
+      announcerString === strings.cameraActionTurnedOffAnnouncement
+        ? strings.cameraActionTurnedOnAnnouncement
+        : strings.cameraActionTurnedOffAnnouncement
+    );
+  }, [announcerString, strings.cameraActionTurnedOffAnnouncement, strings.cameraActionTurnedOnAnnouncement]);
+
   const onToggleClick = useCallback(async () => {
     // Throttle click on camera, need to await onToggleCamera then allow another click
     if (onToggleCamera) {
+      // allows for the setting of narrator strings triggering the announcer when camera is turned on or off.
       setWaitForCamera(true);
       try {
         await onToggleCamera(localVideoViewOptions ?? defaultLocalVideoViewOptions);
       } finally {
         setWaitForCamera(false);
+        toggleAnnouncerString();
```

Comment:
anything in the `finally` block will always execute, (whether the try block threw an error or not), but if onToggleCamera fails we won't want to update the announcer string -- can this be placed after the `await onToggleCamera` inside the try catch?

Reply:
yes will move 😊